### PR TITLE
Launch ros_gz_bridge from xml

### PR DESCRIPTION
**Pull request by caguero**
# 🎉 New feature

Part of https://github.com/gazebosim/ros_gz/issues/552

## Summary

This patch allows to run the `ros_gz` bridge as an executable or as a composable node from an XML file.

### How to test it?

Use the provided `ros_gz_bridge.launch` (modify `config_file` accordingly) to run the bridge as an executable

```
ros2 launch ros_gz_bridge ros_gz_bridge.launch config_file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_bridge/test/config/full.yaml
```

Verify that you see the `/ros_chatter` topic:
```
caguero@cold:~/ros_gz_ws$ ros2 topic list
/parameter_events
/ros_chatter
/rosout
```

And confirm that no container has been created:
```
caguero@cold:~/ros_gz_ws$ ros2 component list

```

Now `CTRL-C` the bridge and try the composable version:
```
 ros2 launch ros_gz_bridge ros_gz_bridge.launch config_file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_bridge/test/config/full.yaml use_composition:=True
```

Verify that you see the `/ros_chatter` topic:
```
caguero@cold:~/ros_gz_ws$ ros2 topic list
/parameter_events
/ros_chatter
/rosout
```

And confirm that a container has been created:
```
caguero@cold:~/ros_gz_ws$ ros2 component list
/ros_gz_container
  1  /ros_gz_bridge
```

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: launch_gz/launch_gz/actions/ros_gz_bridge.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    def __init__(
+        self,
+        *,
+        config_file: SomeSubstitutionsType,
+        container_name: SomeSubstitutionsType,
+        namespace: SomeSubstitutionsType,
+        use_composition: SomeSubstitutionsType,
+        use_respawn: SomeSubstitutionsType,
+        log_level: SomeSubstitutionsType,
```
Comment (azeey) [inline launch_gz/launch_gz/actions/ros_gz_bridge.py]: All of these parameters are optional in `ros_gz_bridge.launch.py` right? Can we do the same here?

Here's an example: https://github.com/ros2/launch_ros/blob/2bf4e6057dea57669c19395f6f39d390bd420ee7/launch_ros/launch_ros/actions/node.py#L121-L129

File: ros_gz_sim/launch/ros_gz_sim.launch [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    use_composition="$(var use_composition)">
+  </gzserver>
+  <ros_gz_bridge 
+    config_file="$(var config_file)"
+    container_name="$(var container_name)"
+    namespace="$(var namespace)"
+    use_composition="$(var use_composition)"
+    use_respawn="$(var use_respawn)"
+    log_level="$(var log_level)">
+  </ros_gz_bridge>
```
Comment (azeey) [inline ros_gz_sim/launch/ros_gz_sim.launch]: Looks very nice!! 🎉

**Review by ahcorde — APPROVED** [review]

File: launch_gz/launch_gz/actions/ros_gz_bridge.py (hunk 1 above)
Comment (caguero) [reply launch_gz/launch_gz/actions/ros_gz_bridge.py]: Done in https://github.com/gazebosim/ros_gz/pull/550/commits/0bfc81dc84635e0b0085f2a4b8f7b928171e02ab.

**Review by azeey — APPROVED** [review]
LGTM!

**Pull request merged**